x-pack/plugins/security_solution/cypress/e2e/dashboards/detection_response.cy.ts

```diff
@@ -23,30 +23,27 @@ import {
   USER_TABLE_ROW_TOTAL_ALERTS,
   USER_TABLE_USER_NAME_BTN,
 } from '../../screens/detection_response';
-import { DETECTION_RESPONSE } from '../../screens/security_header';
 import { QUERY_TAB_BUTTON, TIMELINE_DATA_PROVIDERS_CONTAINER } from '../../screens/timeline';
 import { waitForAlerts } from '../../tasks/alerts';
 import { createRule } from '../../tasks/api_calls/rules';
 import { cleanKibana } from '../../tasks/common';
 import { investigateDashboardItemInTimeline } from '../../tasks/dashboards/common';
 import { waitToNavigateAwayFrom } from '../../tasks/kibana_navigation';
 import { login, visit } from '../../tasks/login';
-import { navigateFromHeaderTo } from '../../tasks/security_header';
-import { closeTimeline } from '../../tasks/timeline';
 import { ALERTS_URL, DASHBOARDS_URL, DETECTIONS_RESPONSE_URL } from '../../urls/navigation';
 
 describe('Detection response view', () => {
   before(() => {
     cleanKibana();
+  });
+
+  beforeEach(() => {
```

Comment:
it's intentional, each `it` should be creating and removing the artifacts after it's done with checking, because if the first `it` would leave some unexpected artfiacts, the rest of test could fail because of that

Reply:
you're right, potentially we could move it to the `before` instead @szwarckonrad

Reply:
Moved back to `before` in 0b6e1b515a9dd4d1ab8112773f9b53311ea87904